ldcbench.nodes/pom.xml

```diff
@@ -14,7 +14,7 @@
   <packaging>jar</packaging>
 
   <properties>
-    <spring.version>5.1.4.RELEASE</spring.version>
+    <spring.version>5.3.6</spring.version>
   </properties>
```

Comment:
Can you please move these spring dependencies into the parent pom and define their versions in `dependencyManagement`? It makes it easier to maintain them in one place instead of having several poms in parallel that have the same library version :wink: